crypto/ec/ecdh_ossl.c

```diff
@@ -66,6 +66,10 @@ int ecdh_simple_compute_key(unsigned char **pout, size_t *poutlen,
     BN_CTX_start(ctx);
     x = BN_CTX_get(ctx);
     y = BN_CTX_get(ctx);
+    if (y == NULL) {
```

Comment:
I do NOT think we should be checking all return values. BN_CTX is not supposed to be used multi-threaded. The pattern to only check the last return value is extensively used and it is unnecessary to check every one.